batch/batch/batch.py

```diff
@@ -1196,6 +1196,7 @@ def is_successful(self):
                                      indent=2)}
 
 
+@routes.get('')
 @routes.get('/')
 @web_authenticated_users_only()
```

Comment:
We should change the name of this function `batch_id`...

Reply:
Agreed, renamed to `index`.